report.md

```diff
@@ -0,0 +1,149 @@
+---
+geometry: margin=2.5cm
+output: pdf_document
+colorlinks: true
+---
+
+<!-- Build PDF using pandoc: $ pandoc report.md -o report.pdf -->
+
+# Report for the NumFOCUS small development grants program
```

Comment:
I would make a reference (in this title or in the first paragraph) to poliastro and the year of the funding (2020 3rd round)

Reply:
Good idea!